wqflask/wqflask/show_trait/SampleList.py

```diff
@@ -162,12 +162,16 @@ def get_first_attr_col(self):
 
         return first_attr_col
 
-def natural_sort(list, key=lambda s: s):
+def natural_sort(a_list, key=lambda s: s):
     """
     Sort the list into natural alphanumeric order.
     """
     def get_alphanum_key_func(key):
         def convert(text): return int(text) if text.isdigit() else text
         return lambda s: [convert(c) for c in re.split('([0-9]+)', key(s))]
     sort_key = get_alphanum_key_func(key)
-    list.sort(key=sort_key)
\ No newline at end of file
+
+    sorted_list=sorted(a_list,key=sort_key)
+
+    return sorted_list
```

Comment:
**issue:** One final change. Please change https://github.com/Alexanderlacuna/genenetwork2/blob/55aa5bf7ec00fb5b5d28a1847e7f8682a62aa547/wqflask/wqflask/show_trait/SampleList.py#L115. Should now be `self.attributes[key].distinct_values = natural_sort(...)`

Reply:
@BonfaceKilz Thanks for catching that